pack/tdc/tdcp.json

```diff
@@ -661,6 +682,24 @@
         "type_code": "asset",
         "xp": 0
     },
+    {
+        "code:": "11036",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "illustrator": "Pixoloid Studios",
+        "name": "Forensic Kit",
+        "pack_code": "tdcp",
+        "position": 36,
+        "quantity": 2,
+        "skill_intellect": 1,
+        "skill_agility": 1,
+        "slot": "Hand",
+        "text": "Uses (4 supplies)\n[action] Spend 1 supply: <b>Investigate.</b> For this investigation, you may use [agility] instead of [intellect], and you get +1 skill value. If you succeed and there is an exhausted enemy at your location, heal 1 horror.",
```

Comment:
```suggestion
        "text": "Uses (4 supplies).\n[action] Spend 1 supply: <b>Investigate.</b> For this investigation, you may use [agility] instead of [intellect], and you get +1 skill value. If you succeed and there is an exhausted enemy at your location, heal 1 horror.",
```